test/models/deploy_spec_test.rb

```diff
@@ -318,14 +318,32 @@ class DeploySpecTest < ActiveSupport::TestCase
       assert_equal 'SAFETY_DISABLED', variable_definition.name
     end
 
-    test "task definitions prepend bundle exec if necessary" do
+    test "task definitions prepend bundle exec if enabled" do
+      Shipit.expects(:automatically_prepend_bundle_exec).returns(true).at_least_once
```

Comment:
Should we also test the default behaviour when `automatically_prepend_bundle_exec` is not set?